geo: failure in TestInverse

The following command fails reliably on my machine:

```
$ make test PKG=./pkg/geo/geographiclib TESTS=TestInverse
[...]
--- FAIL: TestInverse (0.00s)
    --- FAIL: TestInverse/{0,0},_{1,1}_on_WGS84Spheroid (0.00s)
        geographiclib_test.go:45:
                Error Trace:    geographiclib_test.go:45
                Error:          Not equal:
                                expected: 45.19676732164486
                                actual  : 45.196767321644856
                Test:           TestInverse/{0,0},_{1,1}_on_WGS84Spheroid
FAIL
FAIL    github.com/cockroachdb/cockroach/pkg/geo/geographiclib  0.017s
```

--- a/pkg/geo/geographiclib/geographiclib_test.go
+++ b/pkg/geo/geographiclib/geographiclib_test.go
@@ -19,6 +19,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const epsilon = 1e-8
+
 func TestInverse(t *testing.T) {
 	testCases := []struct {
 		desc     string
@@ -40,9 +42,9 @@ func TestInverse(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.desc, func(t *testing.T) {
 			s12, az1, az2 := tc.spheroid.Inverse(tc.a, tc.b)
-			require.Equal(t, tc.s12, s12)
-			require.Equal(t, tc.az1, az1)
-			require.Equal(t, tc.az2, az2)
+			require.InEpsilon(t, tc.s12, s12, epsilon)
+			require.InEpsilon(t, tc.az1, az1, epsilon)
+			require.InEpsilon(t, tc.az2, az2, epsilon)
 		})
 	}
 }
@@ -69,7 +71,7 @@ func TestInverseBatch(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.desc, func(t *testing.T) {
 			sum := tc.spheroid.InverseBatch(tc.points)
-			require.Equal(t, tc.sum, sum)
+			require.InEpsilon(t, tc.sum, sum, epsilon)
 		})
 	}
 }
@@ -98,8 +100,8 @@ func TestAreaAndPerimeter(t *testing.T) {
 	for _, tc := range testCases {
 		t.Run(tc.desc, func(t *testing.T) {
 			area, perimeter := tc.spheroid.AreaAndPerimeter(tc.points)
-			require.Equal(t, tc.area, area)
-			require.Equal(t, tc.perimeter, perimeter)
+			require.InEpsilon(t, tc.area, area, epsilon)
+			require.InEpsilon(t, tc.perimeter, perimeter, epsilon)
 		})
 	}
 }

